plugins/builds/getBuildStatuses.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const stringSchema = joi.string().regex(/^[0-9]+$/);
```

Comment:
why string? use joi.number().positive()

Reply:
Because the parameters are from request.query they all come in as strings. Regardless of how they are passed in when calling the API they are converted to string when they are sent.

Reply:
don't we have anything existing in `data-schema` ?